docs/usage.md

```diff
@@ -1,43 +1,81 @@
+2. Takes the indicator value from the command to create an indicator in FortiSOAR&trade; with the information specified in the input form on the MS Teams app.
+
+### Syntax
+
+```
+@fortiSOAR createIndicator <indicator_value>
+```
 
-    `@fortiSOAR createIndicator <indicator_value>` adds an indicator in FortiSOAR and gets the latest enrichment details back to Microsoft Teams within seconds.
+### Example
 
-    For example, `@fortiSOAR createIndicator gumblar.cn` creates an indicator of value `gumblar.cn`.
+```
+@fortiSOAR createIndicator gumblar.cn
+```
+
+This command adds an indicator of value `gumblar.cn` in FortiSOAR&trade;
 
 You can create indicators on the fly for enrichment and further investigation by the Security Operations Center (SOC) team.
 
+For more information and examples for this command, refer to the [Adding an indicator with the indicator value specified](https://docs.fortinet.com/document/fortisoar/1.0.0/fortisoar-for-microsoft-teams-application/630/fortisoar-for-microsoft-teams-application-v1-0-0#Adding_an_indicator_with_the_indicator_value_specified) section of FortiSOAR&trade; product documentation.
+
 ## `@fortiSOAR invokePlaybook`
 
-This command takes another argument `<tag_name>` to trigger the playbook containing the tag `<<tag_name>>`.
+This command takes another argument `<tag_name>` to trigger the playbook containing the tag `<tag_name>`.
+
+> **NOTE**:  If there are multiple playbooks with the same tag, the first found playbook is executed.
+
+To create playbooks that can be triggered from Microsoft Teams chat interface, refer to [Create a custom playbook to be triggered from Microsoft Teams](https://docs.fortinet.com/document/fortisoar/1.0.0/fortisoar-for-microsoft-teams-application/630/fortisoar-for-microsoft-teams-application-v1-0-0How_to_create_a_custom_playbook_that_can_be_triggered_from_Microsoft_Teams_) section of FortiSOAR&trade; product documentation.
 
-For example, the command `invokePlaybook getIPRep` invokes playbooks with the tag `getIPRep`.
+You can design the manual input to send interactive input forms inline on Microsoft Teams chat interface. Refer to this [example of running a Manual Input playbook that uses Microsoft Teams as a delivery medium](https://docs.fortinet.com/document/fortisoar/1.0.0/fortisoar-for-microsoft-teams-application/630/fortisoar-for-microsoft-teams-application-v1-0-0#Example_of_running_a_Manual_Input_playbook_that_uses_Microsoft_Teams_as_a_delivery_medium).
 
-This powerful action opens the automation capabilities for FortiSOAR where you can create and invoke playbooks in FortiSOAR.
+### Syntax
 
->**NOTE**: The playbook you are trying to invoked must have an existing `bot_enabled` tag. 
+```
+@fortiSOAR invokePlaybook <tag_name>
+```
+
+### Example
+
+```
+@fortiSOAR invokePlaybook getIPRep
+```
+
+The command invokes the playbooks with the tag `getIPRep`.
+
+>**NOTE**: The playbook being invoked must have a `bot_enabled` tag.
```

Comment:
The playbook being invoked must have a `bot_enabled` tag.  --> The playbook being invoked must ALSO have a `bot_enabled` tag.